client/index.html

```diff
@@ -276,23 +274,89 @@ <h4 id="totals-hot-name"></h4>
           var totals_table = tables[tables.length - 1];
           return function (changes, source) {
             if (changes == null) return;
+            
             for (var i = 0; i < changes.length; i++) {
               var change = changes[i];
               var old = change[2];
               var val = change[3];
-
-              old = (old == '' || old == null) ? 0 : old;
-              val = (val == '' || val == null) ? 0 : val;
-              if (old == val) continue;
-
+              
```

Comment:
I use gedit. I just saw the .editorconfig file. I will get the plugin and fix the .editconfig and the spaces after if/for/etc. When I am back to the machine tomorrow.